src/v2/view-builder/views/TerminalView.js

```diff
@@ -2,9 +2,10 @@ import { createCallout, loc } from 'okta';
 import BaseView from '../internals/BaseView';
 import BaseForm from '../internals/BaseForm';
 import BaseFooter from '../internals/BaseFooter';
-import {getBackToSignInLink} from '../utils/LinksUtil';
+import { getBackToSignInLink, getSkipSetupLink } from '../utils/LinksUtil';
 
 const RETURN_LINK_EXPIRED_KEY = 'idx.return.link.expired';
+const SAFE_MODE_KEY = 'idx.error.server.safe.mode';
```

Comment:
we should add some comments in the API code to know this should be the prefix for all error messages for safe mode
@anishashetty-okta

Reply:
If we rely on the fact that just skip remediation -> safeMode then we dont have to worry about this.

Reply:
Ack. will update my PR

Reply:
@magizh-okta we may or may not have skip remediation.. our only signal is the error message key